### PR TITLE
Version bump: update copyright year

### PR DESCRIPTION
**Pull request by earboxer**
Also adds minimal documentation for ssml pitch and range that I should have done in 2018.

### CODE REVIEW

File: main/mimic_main.c [hunk 1]
```diff
@@ -78,7 +78,7 @@ void sigint_handler(int signum)
 static void mimic_version()
 {
     printf("  Carnegie Mellon University, Copyright (c) 1999-2011, all rights reserved\n");
-    printf("  mimic developers, Copyright (c) 2016, all rights reserved\n");
+    printf("  mimic developers, Copyright (c) 2016-2020, all rights reserved\n");
     printf("  version: %s-%s (%s)\n",
            PACKAGE_NAME, PACKAGE_VERSION, PACKAGE_URL);
```
Comment (earboxer) [inline main/mimic_main.c]: The PACKAGE_URL is not being set when I build. Should this be set to something? (or maybe just removed?)

Comment (forslund) [reply main/mimic_main.c]: It doesn't show for me either... Looking at the url and the [ac_init documentation](https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Initializing-configure.html) it seems like it's the PACKAGE_BUGREPORT we add. But we don't use it.

I say remove it. The bugreport url might be good to have in the help but feels misplaced when checking the version.

**Comment by earboxer** [comment]
I made the changes. Now it looks like
```sh
./mimic --version
If audio works ignore messages below
If audio works ignore messages above
  Carnegie Mellon University, Copyright (c) 1999-2011, all rights reserved
  mimic developers, Copyright (c) 2016-2020, all rights reserved
  version: mimic-1.3.0.1
./mimic --help
If audio works ignore messages below
If audio works ignore messages above
mimic: a small simple speech synthesizer
  Carnegie Mellon University, Copyright (c) 1999-2011, all rights reserved
  mimic developers, Copyright (c) 2016-2020, all rights reserved
  version: mimic-1.3.0.1
Report bugs to https://github.com/MycroftAI/mimic/issues.
usage: mimic TEXT/FILE [WAVEFILE]
```

(I'm just going to pretend that "if audio works" message... it sends to stderr anyway, so not a big deal).

**Comment by earboxer** [comment]
Note that the pipeline is failing (for ios builds)

> No package 'libpcre2-8' found

Supposedly we allow these failures: 68fa5326f9402e41721ff74e00cb36b1beecf76f

**Comment by forslund** [comment]
Thanks @earboxer, this looks good. Merging.

**Comment by forslund** [comment]
I have to see why the test I thought was disabled showed an error here. Or better figure out what's wrong with the ios build...

**Comment by earboxer** [comment]
Thanks! Can you add a [tag](https://github.com/MycroftAI/mimic1/releases) for this release?

**Comment by forslund** [comment]
Will tag it tomorrow morning when I'm at a computer.

Den sön 16 feb. 2020 22:27Zach DeCook <notifications@github.com> skrev:

**Comment by earboxer** [comment]
I hate to sound like a broken record... (maybe the tag didn't get pushed?).

**Comment by forslund** [comment]
Could have sworn I did this...

You are totally within your rights to nag. Will do asap!

**Pull request merged**